Library/Homebrew/dev-cmd/man.rb

```diff
@@ -56,11 +56,47 @@ def path_glob_commands(glob)
     end.reject { |s| s.strip.empty? || s.include?("@hide_from_man_page") }
   end
 
+  def path_glob_commands_from_DSL(glob)
+    all = []
+    Pathname.glob(glob)
+      .sort_by { |source_file| sort_key_for_path(source_file) }
+      .map do |source_file|
+        cmd = Pathname(source_file).basename(".rb")
+        if cmd.fnmatch?("*.sh")
+          next
+        end
+        require "cmd/#{cmd}"
+        class_name = cmd.to_s.gsub(/^--/, '').gsub(/-/, '_')
```

Comment:
This is because legal class names cannot begin with ```--``` nor ```__``` and nor can they have ```-``` in the name. So, commands such as ```tap-pin``` will have class name ```Tap_pinCommands``` and commands such as ```--cache``` will have class names as ```CacheCommands```